components/src/dynamo/profiler/utils/dgdr_v1beta1_types.py

```
@@ -287,6 +287,14 @@ class DynamoGraphDeploymentRequestSpec(BaseModel):
         default=True,
         description="AutoApply indicates whether to automatically create a DynamoGraphDeployment after profiling completes. If false, the generated spec is stored in status for manual review and application.",
     )
+    envs: Optional[List[corev1.EnvVar]] = Field(
```

Ruff (0.15.12)

[error] Undefined name `corev1`

(F821)